client/introduction.mdx

```diff
@@ -111,66 +118,71 @@ All Pipecat client SDKs provide:
     });
 
     try {
-      await rtviClient.connect();
+      // you can provide your Daily connection credentials directly as
+      // part of connect or provide a REST endpoint for acquiring them
+      await pcClient.connect({
+        endpoint: "https://your-connect-end-point-here/connect",
+      });
     } catch (e) {
       console.error(e.message);
     }
 
-    // Events
-    rtviClient.on(RTVIEvent.Connected, () => {
+    // Alternative Event Handling Setup
+    pcClient.on(RTVIEvent.Connected, () => {
       console.log("[EVENT] User connected");
     });
-    rtviClient.on(RTVIEvent.Disconnected, () => {
+    pcClient.on(RTVIEvent.Disconnected, () => {
       console.log("[EVENT] User disconnected");
     });
     ```
 
+    ```react react
+    // Example: Using PipecatClient in a React component
+    ```
+
+    </CodeGroup>
+
 </Tab>
-<Tab title="Single-turn Actions">
-  Send individual actions for:
-  - Text-to-text chat
-  - One-off image processing
-  - Batch operations
-
-```javascript
-// Example: Sending a single action
-import { RTVIClient } from "realtime-ai";
-
-const rtviClient = new RTVIClient({
-  params: {
-    baseUrl: "https://your-connect-end-point-here",
-    endpoint: {
-      connect: "/connect",
-      action: "/action",
-    },
-  },
-  callbacks: {
-    onBotText: (text: BotLLMTextData) => {
-      console.log("Text response:", text);
-    },
-  },
-});
-
-try {
-  await rtviClient.action({
-    service: "llm",
-    action: "append_to_messages",
-    arguments: [
-      {
-        name: "messages",
-        value: [
-          {
-            role: "user",
-            content: "tell me a joke",
-          },
-        ],
+<Tab title="Custom Messaging">
```

Comment:
Philosophical question: if we're no longer selling Pipecat Client as a tool for sending "non-custom" bot configuration messages (we're recommending folks do their bot configuration server-side), then can we just call this "messaging" as opposed to "custom messaging"?

Reply:
i think i had also planned on maybe adding more tabs for any other most-common patterns, but only did messaging for now as a replacement for what was here, which was an "actions" example.